frontend/src/component/feature/Dependencies/AddDependency.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, styled } from '@mui/material';
+import { trim } from '../../common/util';
+import React, { FC, useState } from 'react';
+import Input from '../../common/Input/Input';
+import { CREATE_FEATURE_STRATEGY } from '../../providers/AccessProvider/permissions';
+import PermissionButton from '../../common/PermissionButton/PermissionButton';
+import { useDependentFeaturesApi } from 'hooks/api/actions/useDependentFeaturesApi/useDependentFeaturesApi';
+
+const StyledForm = styled('form')({});
+
+const StyledInputDescription = styled('p')(({ theme }) => ({
+    marginBottom: theme.spacing(1),
+}));
+
+const StyledInput = styled(Input)(({ theme }) => ({
+    marginBottom: theme.spacing(2),
+}));
+
+interface IAddDependencyProps {
+    projectId: string;
+    featureId: string;
+}
+export const AddDependency: FC<IAddDependencyProps> = ({
```

Comment:
this entire component will be gone once we have proper design. This is just for manual testing purposes